application/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Float,
+    ForeignKey,
+    DateTime
+)
+from sqlalchemy.orm import relationship
+from application.database import Base
+
+
+class City(Base):
+    __tablename__ = "cities"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, index=True)
+    additional_info = Column(String, nullable=True)
+
+
+class Temperature(Base):
+    __tablename__ = "temperatures"
+
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer, ForeignKey("cities.id"))
```

Comment:
Consider setting `nullable=False` for the `city_id` column to ensure that every temperature record is associated with a city, if this is a requirement for your application.